cwms_radar_api/src/main/java/cwms/radar/api/ClobController.java

```diff
@@ -260,14 +262,17 @@ public Clob deserialize(String body, String formatHeader) throws JsonProcessingE
 
     private static ObjectMapper getObjectMapperForFormat(String format) {
         ObjectMapper om;
-        if ((Formats.XMLV2).equals(format)) {
+        log.atInfo().log("Getting objectmapper for format '%s'",format);
+        if (ContentType.equivalent(Formats.XMLV2,format)) {
             JacksonXmlModule module = new JacksonXmlModule();
             module.setDefaultUseWrapper(false);
             om = new XmlMapper(module);
-        } else if (Formats.JSONV2.equals(format)) {
+        } else if (ContentType.equivalent(Formats.JSONV2,format)) {
             om = JsonV2.buildObjectMapper();
         } else {
-            throw new FormattingException("Format is not currently supported for Levels");
+            FormattingException fe = new FormattingException("Format is not currently supported for Levels");
```

Comment:
message should be for Clob not Levels

Reply:
Do'h, I had caught one other of those but missed that one.

Reply:
one of these days we should really move those to a properties file or something, like 90% of that text is duplicate in several controllers.